translations/messages+intl-icu.en.xlf

```diff
@@ -438,6 +438,63 @@
                 <source>paginator.current</source>
                 <target>(current)</target>
             </trans-unit>
+
+            <trans-unit id="action.forgot_password">
+                <source>action.forgot_password</source>
+                <target>Forgot Password</target>
+            </trans-unit>
+            <trans-unit id="reset_password.title">
+                <source>reset_password.title</source>
+                <target>Reset your password</target>
+            </trans-unit>
+
+            <trans-unit id="reset_password.check_email.p1">
+                <source>reset_password.check_email.p1</source>
+                <target>An email has been sent that contains a link that you can click to reset your password. This link will expire in {token} hour(s).</target>
```

Comment:
```suggestion
                <target>An email has been sent that contains a link that you can click to reset your password. This link will expire in {token} {token, plural, =0 {hours} one {hour} other {hours}}.</target>
```
Am I doing this pluralization thing right?

Reply:
no need for `=0`. The pluralization will already know that `0` uses `other` in English.